constants/config.ts

```diff
@@ -1,16 +1,31 @@
 import { i18n } from '../utils/lang';
+import path from 'path';
+import os from 'os';
 
 export const DEFAULT_HUBSPOT_CONFIG_YAML_FILE_NAME = 'hubspot.config.yml';
 
 export const HUBSPOT_CONFIGURATION_FOLDER = '.hscli';
 export const HUBSPOT_CONFIGURATION_FILE = 'config.yml';
 
+export const GLOBAL_CONFIG_PATH = path.join(
+  os.homedir(),
+  HUBSPOT_CONFIGURATION_FOLDER,
+  HUBSPOT_CONFIGURATION_FILE
+);
+
 export const DEFAULT_ACCOUNT_OVERRIDE_FILE_NAME = '.hsaccount';
 export const DEFAULT_ACCOUNT_OVERRIDE_ERROR_INVALID_ID =
   'DEFAULT_ACCOUNT_OVERRIDE_ERROR_INVALID_ID';
 export const DEFAULT_ACCOUNT_OVERRIDE_ERROR_ACCOUNT_NOT_FOUND =
   'DEFAULT_ACCOUNT_OVERRIDE_ERROR_ACCOUNT_NOT_FOUND';
 
+export const DEFAULT_CMS_PUBLISH_MODE = 'defaultCmsPublishMode';
+export const HTTP_TIMEOUT = 'httpTimeout';
+export const ENV = 'env';
+export const HTTP_USE_LOCALHOST = 'httpUseLocalhost';
+export const DEFAULT_ACCOUNT = 'defaultAccount';
```

Comment:
I wonder if we want to make the decision for the user in the case where there's a conflict between `defaultAccount` and `defaultPortal`. We never want `defaultPortal` in the global config because it's deprecated.

I'm realizing a similar thing applies to `accounts` vs. `portals`. Let me know if this migration logic is already handling it though

Reply:
The migration logic handles both, so the properties `defaultPortal` and `portals` will never appear in the new config. 

We do prompt the user if the values of `defaultAccount` and `defaultPortal` are different, and the user has the option to write over the value of `defaultAccount` but not the property name. Let me know if that clears things up, happy to talk it through in a huddle.